change/@fluentui-react-cards-fe2a3d3c-a627-4d49-9716-871901a8b44c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
Should this also be "none"?

Reply:
Good catch, that's a leftover from a previous version of the change

Reply:
I take that back, this actually WAS supposed to be prerelease. Since the webpack bundle is part of the published package, technically removing the bundle should be a version bump.